web/src/admin/enterprise/EnterpriseLicenseListPage.ts

```diff
@@ -1,4 +1,3 @@
-import "@goauthentik/admin/enterprise/EnterpriseLicenseForm";
```

Comment:
What's the reason for removing this? The form is still required by this page

Reply:
The "reason" is a fat finger error.  I didn't mean to remove it.